stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -36,10 +38,11 @@ public static synchronized NetworkPeerManager getOrCreateInstance(Context contex
     return sInstance;
   }
 
-  public NetworkPeerManager(ResponseBodyFileManager responseBodyFileManager) {
+  public NetworkPeerManager(
+      ResponseBodyFileManager responseBodyFileManager) {
     mResponseBodyFileManager = responseBodyFileManager;
+    mPrettyPrinterInitializer = new DefaultPrettyPrinterInitializer();
```

Comment:
related to my comment about removing the `DefaultPrettyPrinterInitializer` class: Another reason why I'd rather see us use `null` is that we're just allocating an object that will either be replaced (via `setPrettyPrinterInitializer`) or that will do nothing. So why not just remove it entirely and use `null` instead :)